docs/source/faq/faq.md

```diff
@@ -67,3 +67,30 @@ Refer to the following table below for a high level guide to each layer's purpos
 | Models         | Stored, serialised pre-trained machine learning models                                                                                                                                                                                                                                                                                                            |
 | Model output   | Analytics specific data model(s) containing the results generated by the model based on the `model input` data                                                                                                                                                                                                                                                    |
 | Reporting      | Reporting data model(s) that are used to combine a set of `primary`, `feature`, `model input` and `model output` data used to drive the dashboard and the views constructed. It encapsulates and removes the need to define any blending or joining of data, improve performance and replacement of presentation layer without having to redefine the data models |
+
+## How can I check the version of Kedro installed?
+To check the version installed, type `kedro -V` in your terminal window.
+
+
+## How do I install a development version of Kedro?
+
+This section explains how to try out a development version of Kedro direct from the [Kedro GitHub repository](https://github.com/kedro-org/kedro).
```

Comment:
Worth pointing directly to the wiki and remove this content from here? https://github.com/kedro-org/kedro/wiki/Guidelines-for-contributing-developers